string game_topic
int32 first_player
string game_type

